<ui version="4.0" >
 <class>AutostartConfig</class>
 <widget class="QWidget" name="AutostartConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>384</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item rowspan="7" row="0" column="0" >
    <widget class="QTreeWidget" name="listCMD" >
     <property name="showDropIndicator" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="btnAddProgram" >
     <property name="text" >
      <string>Add Program...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="btnAddScript" >
     <property name="text" >
      <string>Add Script...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="btnRemove" >
     <property name="text" >
      <string>&amp;Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="btnProperties" >
     <property name="text" >
      <string>&amp;Properties...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>79</width>
       <height>141</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QPushButton" name="btnAdvanced" >
     <property name="text" >
      <string>Advanced...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
